### DISCUSSION TITLE
Connection Issues IPv6

### DISCUSSION THREAD

**Discussion by mahescho** [rustdesk/rustdesk#9275] [Q&A]
At one site the internet connection changed and it now is IPv6 wihe CGN for IPv4. My RustDesk Server is reachable be IPv4 and IPv6. When I connect to a client on the IPv6 only site it connects but no image gets displayed.

How to fix this?

**Comment by basilgello** [comment]
Without logs, gard to say: https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs

Just redact your sensitive info
